### DISCUSSION TITLE
Is rawUser made by pocketbase or the provider?

### DISCUSSION THREAD

**Discussion by CogalTek** [pocketbase/pocketbase#3725] [Q&A] [answered]
Hello, I would like to know if the rawUser object is an object created by pocketbase or if it is the direct response coming from a provider?

because I'm missing information like the email address in the Microsoft one and there is also a token, I don't know what this token is for

can you help me ? thank you

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, the `rawUser` field is the direct OAuth2 user profile response and it will depend on the OAuth2 app configurations and scopes.

To enable returning the email, you could check this comment - https://github.com/pocketbase/js-sdk/issues/176#issuecomment-1518939458.

**Reply by CogalTek** [reply]
ok thank you i will see that
